configuration/configuration.go

```diff
@@ -120,8 +120,9 @@ const (
 	//
 	//------------------------------------------------------------------------------------------------------------------
 
-	varAccessTokenExpiresIn  = "useraccount.token.access.expiresin"  // In seconds
-	varRefreshTokenExpiresIn = "useraccount.token.refresh.expiresin" // In seconds
+	varAccessTokenExpiresIn    = "useraccount.token.access.expiresin"    // In seconds
```

Comment:
or `...expires.in`?

Reply:
I haven't changed this variable, it's only showing here because of reformatting